src/app/pages/OpenWalletPage/Features/FromMnemonic/index.tsx

```diff
@@ -13,19 +13,19 @@ export function FromMnemonic(props: Props) {
   const walletActions = useWalletSlice().actions
   const dispatch = useDispatch()
 
-  const [mnemonic, setMnemonic] = React.useState('')
+  const [rawMnemonic, setRawMnemonic] = React.useState('')
   const [mnemonicIsValid, setMnemonicIsValid] = React.useState(true)
 
+  const mnemonic = rawMnemonic.trim().replace(/\n+/g, ' ').replace(/ +/g, ' ')
```

Comment:
could we do this in a single `.replace(/[ \n]+/g, ' ')`

Reply:
Sure :)